src/core/random.hpp

```diff
@@ -78,51 +78,55 @@ void init_random_seed(int seed);
 } /* Random */
 
 /**
- * @brief Draws a random real number from the uniform distribution in the range [0,1)
+ * @brief Draws a random real number from the uniform distribution in the range
+ * [0,1)
 */
 
 inline double d_random() {
   using namespace Random;
-  return uniform_real_distribution(generator); 
+  return uniform_real_distribution(generator);
 }
 
 /**
- * @brief draws a random integer from the uniform distribution in the range [0,maxint-1]
+ * @brief draws a random integer from the uniform distribution in the range
+ * [0,maxint-1]
  *
  * @param maxint range.
  */
-inline int i_random(int maxint){
+inline int i_random(int maxint) {
   using namespace Random;
-  std::uniform_int_distribution<int> uniform_int_dist(0, maxint-1);
+  std::uniform_int_distribution<int> uniform_int_dist(0, maxint - 1);
   return uniform_int_dist(generator);
 }
 
 /**
- * @brief draws a random number from the normal distribution with mean 0 and variance 1.
+ * @brief draws a random number from the normal distribution with mean 0 and
+ * variance 1.
  */
-inline double gaussian_random(void){
+inline double gaussian_random(void) {
   using namespace Random;
   return normal_distribution(generator);
 }
 
-
 /**
  * @brief Generator for cutoff Gaussian random numbers.
  *
- * Generates a Gaussian random number and generates a number between -2 sigma and 2 sigma in the form of a Gaussian with standard deviation sigma=1.118591404 resulting in 
+ * Generates a Gaussian random number and generates a number between -2 sigma
+ * and 2 sigma in the form of a Gaussian with standard deviation
+ * sigma=1.118591404 resulting in
  * an actual standard deviation of 1.
  *
  * @return Gaussian random number.
  */
-inline double gaussian_random_cut(void){
+inline double gaussian_random_cut(void) {
   using namespace Random;
-  const double random_number=1.042267973*normal_distribution(generator);
-  
-  if ( fabs(random_number) > 2*1.042267973 ) {
-    if ( random_number > 0 ) {
-      return 2*1.042267973;
+  const double random_number = 1.042267973 * normal_distribution(generator);
```

Comment:
What is this number? (`1.042267973`)  It keeps appearing here but there is no documentation whatsoever explaining the purpose and origin of it.  It was originally introduced in https://github.com/espressomd/espresso/commit/8185eae2d29463bbd7a649617ea9d662d2f821a8 also without docs.